Procfile

```
@@ -0,0 +1,2 @@
+web: bundle exec puma -C config/puma.rb
```

```suggestion
web: bundle exec puma
```
The `config/puma.rb` is a default path and you can omit it.